src/cli/index.ts

```diff
@@ -67,6 +68,7 @@ polykey
   .addCommand(makeKeyManagerCommand())
   .addCommand(makeSecretsCommand())
   .addCommand(makeVaultsCommand())
+  .addCommand(makePeersCommand())
```

Comment:
Oh I now realize you already are grouping the commands. But in that case peer stuff should be further separated.

1. key node peer commands
2. social commands
3. nat commands

We can make them still at the top-level, I think they are sufficiently separate concepts.

Reply:
yeah what we can do is separate them out into domain folders and have separate files (we discussed this before but I just haven't gotten around to doing this)
so folders like `peers` or `secrets`

Reply:
these commands have been separated as described above